clients/go/client.go

```diff
@@ -0,0 +1,21 @@
+package raccoon
+
+type (
+	Event struct {
+		Type string
+		Data interface{}
+	}
+
+	Response struct {
+		Status   int32
+		Code     int32
+		SentTime int64
+		Reason   string
+		Data     map[string]string
+	}
+)
+
+type Client interface {
+	// Send sends a request to raccoon with the message provided.
+	Send([]*Event) (string, *Response, error)
```

Comment:
@AkbaraliShaikh What is the first param string represents here?

Reply:
If we are not using named parameter then we should explain it through comments for client users.

Reply:
okay, i thought the types are self explanatory expect the `string`,
np, I will update

Reply:
yes, the string is not clear, rest two are good.

Reply:
updated